core/client/fs/src/main/java/alluxio/util/FileSystemOptions.java

```diff
@@ -255,6 +255,19 @@ public static SetAttributePOptions setAttributeDefaults(AlluxioConfiguration con
         .build();
   }
 
+  /**
+   * Sets the defaults for the SetAttribute RPC which should be used on the client side.
+   * @param conf Alluxio configuration
+   * @return options based on the configuration
+   */
+  public static SetAttributePOptions setAttributeClientDefaults(AlluxioConfiguration conf) {
+    return SetAttributePOptions.newBuilder()
```

Comment:
can we add a comment or javadoc being explicit that we are only setting sync interval?

Reply:
moved an old comment